src/main/scala/com/gu/subscriptions/cas/directives/HealthCheckDirective.scala

```diff
@@ -1,21 +1,16 @@
 package com.gu.subscriptions.cas.directives
 
-import com.gu.subscriptions.cas.config.Configuration.appConfig
+import com.gu.subscriptions.cas.config.Configuration.nullSettings
 import spray.http.MediaTypes._
 import spray.routing.{HttpService, Route}
 
 trait HealthCheckDirective {this: HttpService =>
-
-  val checkConfig = List(
-    "proxy"
-  ).filter(appConfig.getString(_) == null)
-
```

Comment:
I just thought that it was a bit weird for `appConfig` to leak outside of the `Configuration` class. So what I did was to mark that value as private and expose the list through `nullSettings`.